ecs-agent/tmds/handlers/taskprotection/v1/handlers/handlers.go

```diff
@@ -414,8 +420,28 @@ func getErrorCodeAndStatusCode(err error) (string, string, int, *string) {
 				err))
 			return awsErr.Code(), msg, http.StatusInternalServerError, nil
 		}
-	} else {
-		logger.Error(fmt.Sprintf("non aws error received: %v", err))
-		return ecserrors.ErrCodeServerException, msg, http.StatusInternalServerError, nil
 	}
+
```

Comment:
please add a comment here that this is the aws-sdk-go-v2 error handling and the above error handling (L407-422) will be removed once we complete our migration.